### ISSUE TITLE
frames bieng drawn on resize

### ISSUE THREAD

**Issue by mashrit** [Bali10050/Darkly#74]
here is a video of what i am trying to say

https://github.com/user-attachments/assets/c14c49d3-a277-4670-90a3-9805f4e85e97

and when the cursor updates the window is frame disappears
same happens with kate

**Comment by Bali10050** [comment]
Does this only happen with blur on?

**Comment by mashrit** [comment]
no even with blur off, frames are drawn

**Comment by Bali10050** [comment]
I don't see it happening in kate, only in dolphin when „Transparent Dolphin view” is enabled

**Comment by Bali10050** [comment]
I'm not sure if this can be fixed, if I understand the code correctly, it achieves transparecy by setting setAutoFillBackground to false, and until that happens, a background is drawn. [By default it's false](https://doc.qt.io/qt-6/qwidget.html#autoFillBackground-prop), and I have no idea what sets it to true

**Comment by mashrit** [comment]
> I don't see it happening in kate

yes, in kate it does not happen on normal resize, but when resizing using a script(e.g. krohnkite) it happens

here is a video

https://github.com/user-attachments/assets/9d2f74da-4a48-4988-b6e0-87a685c8172d

**Comment by DeltaCopy** [comment]
@mashrit are you using a custom custom color scheme or one that can be downloaded from kde.store ?
For testing purposes, can you please share the link or contents of the scheme here.

**Comment by mashrit** [comment]
@DeltaCopy yes I do use a custom theme(modified version of [cantadark](https://store.kde.org/p/1306412)

here, the scheme
[Cantadark.colors.zip](https://github.com/user-attachments/files/18237775/Cantadark.colors.zip)

**Comment by DeltaCopy** [comment]
See if https://github.com/Bali10050/Darkly/pull/86 fixes things.
I have tested using the provides the color scheme.
Resizing Dolphin specifically doesn't produce any flickering on my end.

Still a draft because it needs further testing.

**Comment by Bali10050** [comment]
#86 seems to work for me, you did some elite level coding there.
Thanks for the support, and happy holidays!

**Comment by mashrit** [comment]
how do you guys build the pull request without updating the original build?

**Comment by Bali10050** [comment]
The easiest way is to download the source code from the source of the pull request, then use `./install.sh`

**Comment by mashrit** [comment]
> The easiest way is to download the source code from the source of the pull request, then use `./install.sh`

yes, that builds the pull, but also deletes the released version

**Comment by mashrit** [comment]
> See if #86 fixes things.

yep, fixed it👍
